freqtrade/optimize/backtesting.py

```diff
@@ -445,11 +445,13 @@ def start(self) -> None:
 
         min_date = None
         max_date = None
+        processed = None
         for strat in self.strategylist:
-            min_date, max_date = self.backtest_one_strategy(strat, data, timerange)
+            min_date, max_date, processed = self.backtest_one_strategy(strat, data, timerange)
```

Comment:
Same as below, causing memory leaks (it's not fully leaking as we know it's still around) - but it'll increase the amount of memory necessary when using `--strategy-list` by at least 1x per strategy.

Since the data is not used anyway, there's no reason to do this.

Reply:
Not needed to, because of btdata from above.